utils/balances.ts

```diff
@@ -98,3 +91,27 @@ export function updateBalance(
       });
     });
 }
+
+export function getFormattedBalance(balance: any, decimals: number) {
+  return formatBalance({
+    number: BigInt(balance.toString().replaceAll(",", "")),
```

Comment:
using bigInt rather than bignumber is the correct and updated approach 👍

Reply:
:)